CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # TimescaleDB Changelog
 
+## 0.9.0 (2018-02-15)
+
+**High-level changes**
+* Support for multiple .so versions, making for smoother updates between
```

Comment:
I would say `Support for multiple extension versions on different databases in the same PostgreSQL instance. This allows different databases to be updated independently and provides for smoother updates between versions...`